cpp/lazperf/vlr.hpp

```diff
@@ -155,6 +155,66 @@ struct LAZPERF_EXPORT wkt_vlr : public vlr
     virtual vlr_header header() const;
 };
 
+
+struct LAZPERF_EXPORT copc_extents_vlr : public lazperf::vlr
+{
+public:
+
+    struct LAZPERF_EXPORT CopcExtent
+    {
+        double minimum;
+        double maximum;
+
+        CopcExtent();
+        CopcExtent(double minimum, double maximum);
```

Comment:
Maybe let's pass those as const ref to avoid unnecessary copies.
```suggestion
        CopcExtent(const double &minimum,const double &maximum);
```

Reply:
Umm, a pointer is 8 bytes. So is a double. No free lunch.